pkg/estimator/server/estimate.go

```diff
@@ -76,6 +76,11 @@ func (es *AccurateSchedulerEstimatorServer) estimateReplicas(
 
 	var res int32
 	replicas, ret := es.estimateFramework.RunEstimateReplicasPlugins(ctx, snapshot, &requirements)
+
+	if ret.IsUnschedulable() {
+		return replicas, nil
+	}
```

Comment:
This is a short path, meaning we have the minimal replica (0) and don't need to run the following checks.
Shall we add a comment on it?

Reply:
Makes sense, I can add this.